src/Workspaces/Core/Portable/Rename/RenameLocations.cs

```diff
@@ -216,6 +218,25 @@ internal static IEnumerable<ISymbol> GetOverloadedSymbols(ISymbol symbol)
             return new SearchResult(locations.ToImmutable(), implicitLocations, referencedSymbols);
         }
 
+        /// <summary>
+        /// Performs the renaming of the symbol in the solution, identifies renaming conflicts and automatically
+        /// resolves them where possible.
+        /// </summary>
+        /// <param name="replacementText">The new name of the identifier</param>
+        /// <param name="nonConflictSymbols">Used after renaming references. References that now bind to any of these
+        /// symbols are not considered to be in conflict. Useful for features that want to rename existing references to
+        /// point at some existing symbol. Normally this would be a conflict, but this can be used to override that
+        /// behavior.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>A conflict resolution containing the new solution.</returns>
+        public Task<ConflictResolution> ResolveConflictsAsync(
```

Comment:
Should this also be `ResolveConflictsInCurrentProcessAsync`?

Reply:
Same for other methods that just forward to an *InCurrentProcessAsync call

Reply:
Changed this, and that naming is gone now.